htdocs/fichinter/stats/index.php

```diff
@@ -247,7 +242,6 @@
 	// Company
 	print '<tr><td align="left">'.$langs->trans("ThirdParty").'</td><td align="left">';
 	if ($mode == 'customer') $filter='s.client in (1,2,3)';
```

Comment:
If we keep this, when an intervention is done at a thirdparty that is only a supplier, the intervention is not reported in stats. May be we should remove all tests if ($mode == 'customer')  ?